Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhin/audit/retrieve/AuditRetrieveEventsUtilTest.java

```diff
@@ -104,7 +104,7 @@ public void testGetQueryAuditEventResponse() {
         assertEquals("QueryAuditEventsResults.RequestMessageId mismatch", obj.getRequestMessageId(), MESSAGE_ID);
         assertEquals("QueryAuditEventsResults.RelatesTo(mismatch", obj.getRelatesTo(), RELATES_TO);
         assertEquals("QueryAuditEventsResults.UserId mismatch", obj.getUserId(), USER_ID);
-        assertEquals("QueryAuditEventsResults.id mismatch", obj.getId(), ID);
+        assertEquals("QueryAuditEventsResults.id mismatch", obj.getId(), ID.longValue());
```

Comment:
AuditRepositoryRecord has id of type Long and QueryAuditEventsResponseType has id of type long and hence the conversion over here.